### DISCUSSION TITLE
Where can I find the value for `[1]`?

### DISCUSSION THREAD

**Discussion by tomodachi94** [nvim-neorocks/lz.n#16] [Q&A] [answered]
I'm currently moving my configuration from Lazy.nvim to **lz.n combined with Home-Manager (for Nix).** Where can I find the value for `[1]`? Could your documentation be a bit more specific about *what* this value is and *where* it can be found?

Thank you for making this :) I'm looking forward to using it.

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

`[1]` is the name of the plugin that is passed to `load(name)` (which defaults to `:h packadd`).
Depending on what you're using to manage your plugins, that could vary, but it's usually the same as the repo name or plugin name.

In rare cases, you need to be careful, because the repo name and plugin name can differ. With Nix, it's best to look at the `pname` attribute of the package. 
For example, for the [vimPlugins.catppuccin-nvim](https://github.com/NixOS/nixpkgs/blob/c00d587b1a1afbf200b1d8f0b0e4ba9deb1c7f0e/pkgs/applications/editors/vim/plugins/generated.nix#L17254) package, the repo is just called `nvim`, but the `pname` is `catppuccin.nvim`. So with Nix/home-manager, you would put `catppuccin.nvim` for `[1]`.

> Could your documentation be a bit more specific about what this value is and where it can be found?

I'll see if I can come up with something 😃
